_docs/tasks/security/multicluster.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Multicluster PKI
+description: Shows how to use Citadel to construct a multicluster PKI.
+
+weight: 90
+
+---
+{% include home.html %}
+
+This task shows how to use Citadel to construct a multicluster PKI.
+This is an alpha feature and it has the following caveats:
+
+* Currently CA certificates revocation is not supported.
+* Cluster identities are not supported.
+* This architecture may be changed in the future.
+
+Citadel can automatically communicate with an upstream Citadel to rotate its signing certificate.
+In this task, you will set up a two-level PKI involving one Citadel as the online root CA and one
+Citadel as the intermediate CA.
+The architecture shown in this task can be extended to have multiple levels of intermediate CAs,
+and each upstream Citadel can provision certificates for many downstream Citadels.
+
+## Before you begin
+
+* In the cluster for the workloads, set up Istio by following the instructions in the
+  [quick start]({{home}}/docs/setup/kubernetes/quick-start.html).
```

Comment:
Do you want to link to multi-cluster setup link? https://preliminary.istio.io/docs/setup/kubernetes/multicluster-install.html

Also I saw that you are using `install/tools/setupMeshEx.sh`, so is it for multi-cluster or service mesh expansion?

Reply:
The multi-cluster installation you mentioned is not integrated with this multicluster PKI, because it is using a _single_ control plane.

`install/tools/setupMeshEx.sh` is originally for the service mesh expansion usage, but here we leverage it as a util to download the key/cert for a service account.